pkg/zip.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gookit/color"
 	"golang.org/x/sync/errgroup"
 
-	"github.com/skuid/tides/pkg/logging"
+	"github.com/skuid/skuid-cli/pkg/logging"
```

Comment:
This version contains a method called ArchiveWithFilterFunc that is breaking the functionality of deploying componentpacks, I fixed two issues related to that in two tickets, if we can include both in this PR we can close one and a half ticket 3787 (https://github.com/skuid/skuid-cli/pull/91/files) and, 3890 (https://github.com/skuid/skuid-cli/pull/98). I tested the rest and it is working as expected.

Reply:
98 PR
![image](https://user-images.githubusercontent.com/30167939/201570464-694f258f-da18-4c50-b7c9-a0e9b1e7bf4a.png)

91 PR
![image](https://user-images.githubusercontent.com/30167939/201570563-a1ea5b9a-b91a-4cf0-b9ec-29a5fcddc6fd.png)

Reply:
I merged the 91 PR branch into this branch, but the 98 PR change is already in there: https://github.com/skuid/skuid-cli/blob/PLIN-3843-Fix_the_cli/pkg/metadata.go?ts=2#L100